src/ol/render/canvas/textreplay.js

```diff
@@ -122,6 +122,61 @@ ol.render.canvas.TextReplay.labelCache_ = new ol.structs.LRUCache();
 
 
 /**
+ * @param {string} fontSpec CSS font spec.
+ * @param {function(boolean)} callback Callback. Will be called with the loading
+ * state of the font when `checkFont()` is called and when loading has finished.
+ */
+ol.render.canvas.TextReplay.checkFont_ = (function() {
+  var loadedFonts = {};
+  var loadingFonts = {};
+  var text = 'wmytzilWMYTZIL@#/&?$%10';
+  var context;
+
+  function isLoading(fontSpec) {
+    if (!context) {
+      context = ol.dom.createCanvasContext2D();
+    }
+    var loading = false;
+    //FIXME fontFamily detection fails for a font name with '[0-9]px ' in it.
+    var fontFamily = fontSpec.split(/[0-9]px /).pop();
```

Comment:
Curious if we can find a more robust way to get the name.  The `font-size` units could also be `em` or `%`.  And could also be a relative (`smaller`) or absolute (`small`) size.

Reply:
We could use the browser's css parser.  E.g.:
```js
var element = document.createElement('div');
element.style.font = fontSpec;
var fontFamily = element.style.fontFamily;
```

Reply:
Maybe something like this:
```js
/**
 * Get the font family from a font spec.
 * @param {string} The CSS font property.
 * @return {string} The font family.
 */
ol.css.getFontFamily = (function() {
  var style = document.createElement('div').style;
  var cache = {};
  return function(fontSpec) {
    if (!(fontSpec in cache)) {
      style.font = fontSpec;
      cache[fontSpec] = style.fontFamily;
    }
    return cache[fontSpec];
  };
})();
```

Reply:
I added `ol.css.getFontFamilies()` in e54f8c9dbe5100de62e033e4a9203b41c7498312.